### ISSUE TITLE
Load the non-mouse annotations made by MGI curators

### ISSUE THREAD

**Issue by ukemi** [geneontology/noctua#763]
Note this is independent of the mouse curation stream, but will be needed if these annotations need to be modified. I suspect this will be the best place for this ticket as one blocker will be identifers that are present in this file that are missing in Noctua.

These are available as a gaf file here:
http://www.informatics.jax.org/downloads/custom/gene_association.mgi_nonmouse

**Comment by ukemi** [comment]
No, actually this has to do with loading this file. Perhaps I put it in the wrong place, but didn't see a better project.

**Comment by kltm** [comment]
Yeah, this is NEO-entity only, so I would suggest a new project (PLANNING) if it's necessary and not covered by a current project.

**Comment by ukemi** [comment]
Done.

**Comment by ukemi** [comment]
The latest discussion about this ticket on the managers' call was to see if Alex could load these into UniProt and not designate them as a load. This way they would become editable by a curator who has access to Protein2GO. @pgaudet am I remembering correctly?

**Comment by pgaudet** [comment]
Hi @ukemi Yes that was my proposal - we need to find out from @alexsign how complicated this is on his end. These would be an internal source- Alex, would we need some 'internal MGI' source?

**Comment by ukemi** [comment]
thanks @pgaudet. I suspect the internal MGI source already exists in some way as @krchristie already has editing capability in Protein2GO.

**Comment by alexsign** [comment]
@pgaudet @ukemi Protein2GO doesn't have MGI internal source, but this will be straightforward to add. @krchristie has Protein2GO account, but it's not associated to any Internal source.

**Comment by ukemi** [comment]
@alexsign, is there any way to just load the annotations and make them look like they were made by @krchristie?

**Comment by alexsign** [comment]
@ukemi yes, this is passible

**Comment by ukemi** [comment]
Great. Let me confirm with @krchristie at our meeting this morning.

**Comment by hdrabkin** [comment]
Confirmed with @krchristie at the GO-MGI meeting this morning

**Comment by krchristie** [comment]
> @pgaudet @ukemi Protein2GO doesn't have MGI internal source, but this will be straightforward to add. @krchristie has Protein2GO account, but it's not associated to any Internal source.

Hi @alexsign - 

This is my complete list of sources:
Sources: GOCN, PRIV, RI, PINC, GDB, CILI 

The source GOCN is the only one I use to add annotations directly into P2GO, when I have needed to add experimental annotations for non-mouse species in order to propagate them in PAINT. 

However, none of these sources are great choices for these sequence orthology based annotations made by MGI curators.

As "MGI" was already an external source, it was not allowed to be an internal source as well. Could we add a source "MGI-SO" for MGI Sequence Orthology so that there won't be any conflict with the existing use of the "MGI" source.

Thanks,

-Karen

**Comment by krchristie** [comment]
Also, @hdrabkin has just reminded me that this mechanism of making experimental annotations for non-mouse species is no longer active at MGI, so loading these experimental annotations based on mouse ISO annotations only needs to happen as a one-time load that can be edited within P2GO. Once these annotations are editable within P2GO, MGI could remove the non-mouse GAF from our site as there would no longer be any need for to update this file and make corresponding updates in P2GO.
